packages/shared/web/src/header/analytics-dropdown/use-analytics-dropdown.tsx

```diff
@@ -0,0 +1,80 @@
+import { SectionLinkProps } from '@notional-finance/mui';
+import { FormattedMessage } from 'react-intl';
+import {
+  DuneIcon,
+  DashboardIcon,
+  DefiLlamaIcon,
+  TokenTerminalIcon,
+} from '@notional-finance/icons';
+
+export const useAnalyticsDropdown = () => {
+  const links: SectionLinkProps[] = [
+    {
+      title: <FormattedMessage defaultMessage={'Notional Dashboard'} />,
+      to: 'https://info.notional.finance/',
+      icon: (
+        <DashboardIcon
+          sx={{
+            fontSize: '1.5rem',
+          }}
+        />
+      ),
+      description: (
+        <FormattedMessage
+          defaultMessage={`Use Notional's custom-built dashboard for full detail`}
+        />
+      ),
+    },
+    {
+      title: <FormattedMessage defaultMessage={'Dune Dashboard'} />,
+      to: 'https://dune.com/PierreYves_Gendron/Notional-V2-Dashboard',
+      icon: (
+        <DuneIcon
+          sx={{
+            fontSize: '1.5rem',
+          }}
+        />
+      ),
+      description: (
+        <FormattedMessage defaultMessage={`Follow Notional's stats on Dune`} />
+      ),
+    },
+    {
+      title: <FormattedMessage defaultMessage={'Defi Llama'} />,
+      to: 'https://defillama.com/protocol/notional',
```

Comment:
@T-Woodward can you decide on what we want to do with this Defi Llama thing?

Reply:
It is apparently moot now: https://twitter.com/0xngmi/status/1637955928887533570?s=20